<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managerWindow</class>
 <widget class="QDialog" name="managerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>461</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Group Manage</string>
   </property>
   <widget class="QLineEdit" name="groupNameBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addGroup">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delGroup">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QListWidget" name="groupListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>221</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="groupMemberList">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteMember">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Member</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>UserManage</string>
   </property>
   <widget class="QListWidget" name="userListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwdBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCanGroupMG">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Can do group manage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCanUserMg">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Can do user manage</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbIsRoot">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Is administrator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addUser">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteUser">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete User</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modifyUser">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Change</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="userInfoBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="addGroupMember">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add to Group</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>600</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usernameBox</tabstop>
  <tabstop>passwdBox</tabstop>
  <tabstop>userInfoBox</tabstop>
  <tabstop>cbCanGroupMG</tabstop>
  <tabstop>cbCanUserMg</tabstop>
  <tabstop>cbIsRoot</tabstop>
  <tabstop>addUser</tabstop>
  <tabstop>modifyUser</tabstop>
  <tabstop>deleteUser</tabstop>
  <tabstop>addGroupMember</tabstop>
  <tabstop>groupNameBox</tabstop>
  <tabstop>addGroup</tabstop>
  <tabstop>delGroup</tabstop>
  <tabstop>deleteMember</tabstop>
  <tabstop>groupMemberList</tabstop>
  <tabstop>groupListWidget</tabstop>
  <tabstop>userListWidget</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
